standalone/src/main/resources/bin/mule.bat

```diff
@@ -115,8 +115,14 @@ rem Customized for Mule
 rem ###############################################################
 
 :console
-"%_WRAPPER_EXE%" -c %_WRAPPER_CONF% %MULE_OPTS%
-goto :eof
+tasklist | find /i "wrapper-windows" >nul
+if not errorlevel 1 (

```

Comment:
What happens with this is that you will unable to run two runtimes in the same env.